NitroxModel/DataStructures/NitroxId.cs

```diff
@@ -17,7 +17,17 @@ public NitroxId()
         {
             guid = Guid.NewGuid();
         }
-        
+
+        /// <summary>
+        /// This is used to "recreate" a NitroxID from a String
+        /// The XML-serializer needs this after deserializing to convert from "Helper-Types" to internal data Types
```

Comment:
Not a fan of "This is used to", how about "Recreate a NitroxID from a String"?. Besides, I don't see a need to explain where this function is currently being used. Seems obvious to me that it's just a useful helper constructor.